src/BigBlueButton.php

```diff
@@ -548,88 +404,101 @@ public function buildUrl($method = '', $params = '', $append = true)
     /**
      * A private utility method used by other public methods to request HTTP responses.
      *
-     * @param string $url
-     * @param string $payload
-     * @param string $contentType
-     *
-     * @return string
-     *
-     * @throws \RuntimeException
+     * @throws BadResponseException|\RuntimeException
      */
-    private function sendRequest($url, $payload = '', $contentType = 'application/xml')
+    private function sendRequest(string $url, string $payload = '', string $contentType = 'application/xml'): string
     {
-        if (extension_loaded('curl')) {
-            $ch = curl_init();
-            if (!$ch) {
-                throw new \RuntimeException('Unhandled curl error: ' . curl_error($ch));
-            }
+        if (!extension_loaded('curl')) {
```

Comment:
I just improved that function by re-grouping its elements. Not sure if this is covered by composer. I would avoid changing it now and keep it to be 100% on the save side ... otherwise to much refactoring at once ...